traffic_portal/app/src/common/modules/form/deliveryService/form.deliveryService.DNS.tpl.html

```diff
@@ -350,17 +350,15 @@ <h3 ng-if="!open()">Previous Value</h3>
                         </label>
                         <div class="col-md-10 col-sm-10 col-xs-12">
                             <div class="checkbox-scroll-container">
-                                <div ng-repeat="capability in requiredCapabilities">
+                                <div ng-repeat="capability in requiredCapabilities track by $index">
                                     <input
-                                            type="checkbox"
-                                            id="capability"
-                                            name="capability"
-                                            ng-model="selectedCapabilities[capability.name]"
-                                            ng-checked="selectedCapabilities[capability.name]"
+                                        type="checkbox"
+                                        id="capability-{{$index}}"
+                                        name="capability-{{$index}}"
+                                        ng-model="selectedCapabilities[capability.name]"
+                                        ng-checked="selectedCapabilities[capability.name]"
```

Comment:
You don't need a `<label>` for every `<input>`, but if it doesn't have one it should have an `aria-label` attribute set. Probably just the name of the Capability will do.